Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN apt-get update \
 WORKDIR /atrope
 
 RUN groupadd -g 1999 python && \
-    useradd -r -u 1999 -g python python
+    useradd -r -m -u 1999 -g python python
```

Comment:
Two minor comments without requiring changes:
* why would the Dockerfile require adding user and group for Python when using `FROM python:3.11-slim`
* I would prefer to have long instead of short options so I don't have to type `man useradd` to review this :)

Reply:
Adding the `python` user to avoid root, that's just being paranoid :)

Reply:
and I moved to the long options